### PR TITLE
Use macro shortcuts for extern "C" declarations in the public API

### PR DESCRIPTION
**Pull request by jpfr**

### CODE REVIEW

**Comment by traversaro** [comment]
Just a small note: technically identifiers that start with a double underscore `__` are reserved according to the C Standard, section 7.1.3 , see https://stackoverflow.com/a/17307796/1379427 .

File: src/server/.#ua_subscription.h [hunk 1]
```diff
@@ -0,0 +1 @@
+jpfr@virtualbox.1542:1533140287
```
Comment (Infinity95) [inline src/server/.#ua_subscription.h]: Seems to be some editor file?

Comment (jpfr) [reply src/server/.#ua_subscription.h]: removed

**Comment by jpfr** [comment]
> technically identifiers that start with a double underscore __ are reserved according to the C Standard

fixed

**Pull request merged**